src/components/Switch/Switch.js

```diff
@@ -111,6 +94,38 @@ class Switch extends React.PureComponent {
   }
 }
 
-export default React.forwardRef((props, ref) => (
-  <Switch innerRef={ref} {...props} />
+const basePropTypes = {
```

Comment:
@dprzybylek small refactor to make default props and prop types of Switch visible in the docs
We should avoid defining it as a static property on a component class.

Reply:
ok 👍